osu.Game/Screens/SelectV2/BeatmapCarousel.cs

```diff
@@ -367,9 +367,11 @@ public void Filter(FilterCriteria criteria)
         private readonly DrawablePool<PanelBeatmap> beatmapPanelPool = new DrawablePool<PanelBeatmap>(100);
         private readonly DrawablePool<PanelBeatmapSet> setPanelPool = new DrawablePool<PanelBeatmapSet>(100);
         private readonly DrawablePool<PanelGroup> groupPanelPool = new DrawablePool<PanelGroup>(100);
+        private readonly DrawablePool<PanelGroupStarDifficulty> starsGroupPanelPool = new DrawablePool<PanelGroupStarDifficulty>(100);
```

Comment:
i'm pretty sure there's a max of 11 panels ever? probably doesn't even need pooling to be honest..